include/libp2p/crypto/secp256k1_provider/secp256k1_provider_impl.hpp

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef LIBP2P_CRYPTO_SECP256K1_PROVIDER_IMPL_HPP
+#define LIBP2P_CRYPTO_SECP256K1_PROVIDER_IMPL_HPP
+
+#include <memory>
+
+#include <openssl/ec.h>
+#include <libp2p/crypto/secp256k1_provider.hpp>
+
+namespace libp2p::crypto::secp256k1 {
+  class Secp256k1ProviderImpl : public Secp256k1Provider {
+   public:
+    /**
```

Comment:
No need to copy comments from interface class. 
Impl class can be without comments, if inherited methods were already commented in parent class

Reply:
Applied